extensions/ArchLinux/modules/skins/vector.less

```
@@ -115,12 +115,31 @@ body.skin-vector-2022 {
     display: block;
   }
 
-  // Align the TOC button and menu
-  #vector-toc-collapsed-button {
-    margin-left: 0;
+  // Fix style and layout of main menu in the left sidebar
+  @media screen {
+    /* note: #vector-main-menu alone is not unique as of MW 1.40 */
+    #vector-main-menu.vector-main-menu {
+      background: @content-background-color;
+      border: @content-border-style;
+      width: auto;
+      margin-left: 0;
+      margin-bottom: 1em;
+    }
   }
-  #mw-panel-toc {
-    left: auto;
+
+  // Fix layout of the TOC menu in the left sidebar
+  @media screen and (min-width: 1000px) {
+    /* note: #mw-panel-toc alone is not specific enouvh */
```

Same as above + typo fix.

```suggestion
    // NOTE: #mw-panel-toc alone is not specific enough
```